extensions/2.0/Vendor/EXT_feature_metadata/schema/class.property.schema.json

```diff
@@ -17,26 +17,20 @@
     "type": {
```

Comment:
Type should have a default of `SINGLE`, right?

Reply:
Done!